spec/controllers/concerns/state_file/eligibility_offboarding_concern_spec.rb

```diff
@@ -7,6 +7,8 @@
     def index
       redirect_to next_path
     end
+
+    def edit; end
```

Comment:
needed since the concern has a before_action on the edit action and if it didn't exist on the mock controller here, it would error